house/house.go

```diff
@@ -379,6 +398,25 @@ func (s *Server) newDevice(in *pb.AddressRequest, home string) error {
 	return nil
 }
 
+func (s *Server) SendNotification(title string, message string, topic string) error {
```

Comment:
warning: exported method Server.SendNotification should have comment or be unexported (golint)